app/helpers/states.rb

```diff
@@ -19,6 +19,10 @@ def self.name_value_pairs
     self.states
   end
 
+  def self.html_safe_names
+    self.states.map(&:first).to_json.html_safe
```

Comment:
Trivial nit: It took me a little while to understand this. I would have found `hash.values.to_json.html_safe` easier to read.